spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/job/support/StepExecutionResourceBuilder.java

```diff
@@ -34,7 +34,7 @@
 public class StepExecutionResourceBuilder {
 
 	static public StepExecutionResource toResource(StepExecution entity) {
-		return new StepExecutionResource(entity.getId(), entity, generateStepType(entity));
+		return new StepExecutionResource(entity.getJobExecution().getId(), entity, generateStepType(entity));
```

Comment:
what path has to do with step execution id? I don't really understand what this is fixing.